### ISSUE TITLE
Better README with links to notebooks and more examples

### ISSUE THREAD

**Issue by v0dro** [symengine/symengine.rb#12]
The current README is wayy too basic for any practical usage.

More examples and notebook links would be great.

**Comment by certik** [comment]
Indeed. We need to improve it. If you want to help, you can send us a pull
request.
